README.md

```diff
@@ -96,6 +96,41 @@ class Point:
   Schema: ClassVar[Type[Schema]] = Schema
 ```
 
+It is also possible to derive all schemas from your own base class
+```python
+from dataclasses import dataclass
+
+import marshmallow
+import marshmallow_dataclass
+
+def camelcase(s):
```

Comment:
If this is just an example, we can use something simpler, like str.upper, in order to avoid having to add a separate function definition.